sleep_for_days/workflows.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+from datetime import timedelta
+from temporalio import workflow
+from dataclasses import dataclass
+from sleep_for_days.activities import send_email, SendEmailInput
```

Comment:
In workflow code we import activity code with the sandbox disabled, as follows:

```
with workflow.unsafe.imports_passed_through():
    from ...
```

It's hard to remember to do it though... would be good to improve that.

Reply:
Ah I noticed that with the other samples, but the `unsafe` scared me so I imported it normally.